components/style/values/specified/mod.rs

```diff
@@ -60,8 +68,14 @@ impl Parse for CSSColor {
             _ => None,
         };
         input.reset(start_position);
+
+        #[cfg(feature = "gecko")]
+        let color = try!(Color::parse(_context, input));
```

Comment:
Can we implement `Parse` for `cssparser::Color` too? That would allow us to avoid this `cfg` I think.